functions/modules/ses.js

```diff
@@ -0,0 +1,34 @@
+const aws = require('aws-sdk');
+const functions = require('firebase-functions');
+
+aws.config.accessKeyId = functions.config().aws.config.access_key_id;
+aws.config.secretAccessKey = functions.config().aws.config.secret_access_key;
+aws.config.region = functions.config().aws.config.region;
+
+const ses = new aws.SES();
```

Comment:
We prefer sendgrid since email is not verified

Reply:
Btw since we are sending a welcome email anyway, do we want to include a `verify` link to prevent spam?
We then will only send email to verified email later.

Reply:
Do you mean adding a new page & API to let users verify?

Reply:
Yes, since we are sending email anyway, put the verification CTA and move the `thank you and you will be notified` to verification page, and let email to be verified for each subscription seems not much of UX issue.
This probably is extra and not required for now though, maybe we want to use the account system later on and just verify once per user